.gitmodules

```diff
@@ -0,0 +1,4 @@
+[submodule "third_party/frameworks"]
+	path = third_party/frameworks
+	url = https://github.com/open-policy-agent/frameworks.git
+	branch = master
```

Comment:
what happens if we have updates in master of frameworks but we haven't updated the dependencies in go.mod of gatekeeper? For example, a new spec update in frameworks/constraint/deploy.

Reply:
The submodule is only here to provide the CRD yamls for constrainttemplate (gomod doesn't let us sync non-go files).

Eventually we may want to cut release branches as it doesn't look like submodules can sync to tags.

Reply:
there is also this method to include non-source files: https://github.com/go-modules-by-example/index/blob/master/012_modvendor/README.md#vendoring-your-module-download-cache

Reply:
oh yea that sucks. so I guess we just need to make sure when we update go.mod for frameworks, we also run `git submodule update --remote` to update these yamls.

Reply:
@sozercan thanks for the modvendor tip, I've added a `vendor` target to the makefile that will let us update the yaml that way, while still having a minimal vendor directory.